### PR TITLE
Adds a java 8 runtime integration test for standard environment.

### LINKED ISSUE #151: add a java 8 runtime integration test

### PR DESCRIPTION
**Pull request by patflynn**
Fixes #151

### CODE REVIEW

**Comment by loosebazooka** [comment]
So, intergration tests aren't actually run on the CI for gradle. I've been running them locally before doing a release. this is something that I'd probably have to set up. Having said that, can you update the formatting?

```
./gradlew goJF
```

should auto format.

**Comment by patflynn** [comment]
fixed.

File: src/integTest/resources/projects/standard-project-java8/src/main/java/HelloServlet.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  @Override
+  public void service(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    resp.setContentType("text/plain");
+
+    // no cache
+    resp.setHeader("Cache-Control", "no-cache, no-store, must-revalidate"); // HTTP 1.1.
+    resp.setHeader("Pragma", "no-cache"); // HTTP 1.0.
+    resp.setHeader("Expires", "0"); // Proxies.
+
+    resp.getWriter().print("Hello from the App Engine Standard Java project.");
```
Comment (loosebazooka) [inline src/integTest/resources/projects/standard-project-java8/src/main/java/HelloServlet.java]: The expected output is `Hello from the App Engine Standard project`, remove the `Java`

See https://github.com/GoogleCloudPlatform/app-gradle-plugin/blob/master/src/integTest/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPluginIntegrationTest.java#L70

Comment (patflynn) [inline src/integTest/resources/projects/standard-project-java8/src/main/java/HelloServlet.java]: fixed

**Comment by loosebazooka** [comment]
okay, I'll run ./gradlew integTest when tomorrow and merge this in.

**Pull request merged**